tools/etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/RestppConnection.java

```diff
@@ -297,8 +297,12 @@ public RestppConnection(String host, Integer port, Boolean secure,
     }
     this.httpClient = builder.build();
 
-    if (this.token == null && this.username != null && this.password != null) {
+    if (this.token == null && this.username != null && this.password != null && this.graph != null ) {
       getToken();
+    } else if (this.token != null){
+      System.out.println(">>> Token exist, use current token");
```

Comment:
Is this how we report errors in other places?

Reply:
I think it's not an error but an option in current connection logic. So this pr is just a hack to help QE resolve current issue. The logic will be redesigned during revamp.